### ISSUE TITLE
Table object edit - changing default value

### ISSUE THREAD

**Issue by honzak-lib** [HeidiSQL/HeidiSQL#526]
## Steps to reproduce this issue

1. Step 1; Select table, go to basic tab to edit its properties
2. Step 2; Click on 'Default' column and type in "Custom' some default value and click ENTER key
3. Then I get... Instead of closing dialog and continuing work on table properties, dialog stays open a table properties are saved. Sometimes I get this behaviour on second try, first try is ok. Like dialog window lose focus while editing default value of some column.

## Current behavior

Instead of closing dialog and continuing work on table properties, dialog stays open a table properties are saved. Sometimes I get this behaviour on second try, first try is ok. Like dialog window lose focus while editing default value of some column.

## Expected behavior

Enter key should trigger 'OK' button and close dialog so I can continue work on table design.

## Possible solution

Key binding?

## Environment

No specialities

* **HeidiSQL version:** 
  
10.1.0.5471

* **Database system and version:** 
  

MySQL 8.0.11

* **Operating system:** 
  
Windows 10 64bit Pro.

**Comment by honzak-lib** [comment]
Yes enter key on keyboard. Looks like first enter is ok, if you try another column, 2nd and further enter goes to save table properties. Hope you can see it from the video:
https://youtu.be/2ogoWUL5uic
Second default value and enter does nothing, looks like its focus is on Save button on table form.
Thanks!

**Comment by ansgarbecker** [comment]
Probably the same issue/cause as #218

**Comment by honzak-lib** [comment]
Except I don't get crash, just something like losing focus.

**Comment by ansgarbecker** [comment]
Just tested again with the current v12.16 release, and with the v13 preview, and this definitely works now as expected.
